src/test/versionbits_tests.cpp

```diff
@@ -92,6 +106,7 @@ class VersionBitsTester
         for (int i = 0; i < CHECKERS; i++) {
             if (InsecureRandBits(i) == 0) {
                 BOOST_CHECK_MESSAGE(checker[i].GetStateFor(vpblock.empty() ? nullptr : vpblock.back()) == THRESHOLD_DEFINED, strprintf("Test %i for DEFINED", num));
+                BOOST_CHECK_MESSAGE(checker_fixed[i].GetStateFor(vpblock.empty() ? nullptr : vpblock.back()) == THRESHOLD_ACTIVE, strprintf("Test %i for ACTIVE (fixed height)", num));
```

Comment:
This test is called `TestDefined`, but we're checking here for THRESHOLD_ACTIVE for the always-active version bit -- not really sure this makes sense?

I guess maybe it makes a certain amount of sense that we want to check that the always-on bit is ACTIVE and not DEFINED, but I think it could use a comment here (and in `TestStarted`/`TestLockedIn`).  Also, I wonder why there's no similar test for ACTIVE state in `TestFailed`?

Reply:
Added to TestFailed.